src/components/__tests__/FormButtons.test.tsx

```diff
@@ -5,12 +5,6 @@ import { initialFormState } from '../FormPage'
 import { FormEvent } from 'react'
 
 describe('FormButtons', () => {
```

Comment:
For å unngå å gjenta Context verdiene kunne man ha en mocked object for values på toppen av filen og utvide/endre den i de ulike testene. F.eks øverst:

```
const mockedContextValues =  {
          setState: jest.fn(),
          states: initialFormState,
          formPageProps: {
            curStep: 1,
            length: 5,
            goBack: jest.fn(),
            onStepChange: jest.fn(),
            handleSubmit: jest.fn(),
            goToNext: jest.fn(),
          },
        }
```
        
        Og i testfilen:
      ```
   render(
      <FormContext.Provider
        value={{
         ...mockedContextValues,
         curStep: 0
        }}
      >
        <FormButtons />
      </FormContext.Provider>
```
    )